src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -199,6 +199,22 @@ public FeedbackSessionAttributes copyFeedbackSession(String newFeedbackSessionNa
 
         return viewableSessions;
     }
+    
+    /**
+     * checks if there is open or published email sent from the course
+     * @param courseId
```

Comment:
Two issues here:
- "Checks if ..." can be omitted as the `@return` param already explains it.
- Again, `@param courseId` has no value add. Add a description to the parameter.